docs/release.rst

```diff
@@ -304,29 +304,17 @@ Edit branch specific settings:
 
 ::
 
-   vim ./branch-config.mk
+   vim .branch-variables.yml
 
 And change content according to comments in the file.
 
-
-Then correct pykickstart version for the new Fedora release by changing all occurrences of
```

Comment:
You have fixed in https://github.com/rhinstaller/anaconda/pull/4523 only one of the occurrences though. The other one is in [dracut/parse-kickstart](https://github.com/rhinstaller/anaconda/blob/master/dracut/parse-kickstart#L44).